### PR TITLE
Mostly UI/navigation changes and minor fixes

### LINKED ISSUE #20: Setting editing a column value to '0' (without quotes) just deletes it
With columnfamily_action.php?action=edit_row, when you put just 0 in a column family. It gets deleted, rather than set to the string '0'.

### PR DESCRIPTION
**Pull request by ghost**
- UI improvements
- Added up-to-date screenshots (feel free to download them to host them on your own servers, if you like)
- Bug fix https://github.com/sebgiroux/Cassandra-Cluster-Admin/issues/20
- Typo fixes

Hope you like it ;)

### CODE REVIEW

File: describe_keyspace.php [hunk 1]
```diff
@@ -74,10 +74,11 @@
 				for ($i = 0; $i < $nb_cfs; $i++) {
 					$one_cf = $describe_keyspace->cf_defs[$i];
 				
-					$vw_vars['cf_def'] = $one_cf;	
+					$vw_vars['cf_def'] = $one_cf;
 					$vw_vars['columnfamily_name'] = $one_cf->name;				
-					$vw_vars['show_edit_link'] = true;					
-					$vw_vars['list_column_families'] .= getHTML('columnfamily_details.php',$vw_vars);
+					$vw_vars['show_edit_link'] = true;
+					$vw_vars['list_column_families'] .= '<tr><td onclick="document.location.href=\'describe_columnfamily.php?keyspace_name=' . $keyspace_name . '&columnfamily_name=' . $vw_vars['columnfamily_name'] . '\'"><a href="describe_columnfamily.php?keyspace_name=' . $keyspace_name . '&columnfamily_name=' . $vw_vars['columnfamily_name'] . '">' . $vw_vars['columnfamily_name'] . '</a></td></tr>';
```
Comment (sebgiroux) [inline describe_keyspace.php]: Would be nice to create a view file for this and call getHTML() instead :)

File: describe_keyspace.php [hunk 2]
```diff
@@ -74,10 +74,11 @@
...
+					// $vw_vars['list_column_families'] .= getHTML('columnfamily_details.php',$vw_vars);
```
Comment (sebgiroux) [inline describe_keyspace.php]: Remove that comment too

File: views/describe_columnfamily.php [hunk 3]
```diff
@@ -10,11 +10,36 @@
...
-	<a href="columnfamily_action.php?action=create_secondary_index&amp;keyspace_name=<?php echo $keyspace_name; ?>&amp;columnfamily_name=<?php echo $columnfamily_name; ?>" class="btn btn-large btn-primary" style="color: #fff; text-decoration: none;">Create Secondary Index</a>
+		<td onclick="document.location.href='describe_columnfamily.php?view=details&amp;keyspace_name=<?php echo $keyspace_name; ?>&amp;columnfamily_name=<?php echo $columnfamily_name; ?>'"><a href="describe_columnfamily.php?view=details&amp;keyspace_name=<?php echo $keyspace_name; ?>&amp;columnfamily_name=<?php echo $columnfamily_name; ?>">Details</a></td>
 <?php endif; ?>
+	</tr>
+</table>
+
+<div class="well">
+
+<?php
+if ($_GET['view'] == 'details' and !$is_read_only_keyspace):
```
Comment (sebgiroux) [inline views/describe_columnfamily.php]: Same issue as the other pull request, we will need an isset here!

File: views/describe_columnfamily.php [hunk 4]
```diff
@@ -47,4 +65,10 @@
 														  'columnfamily_name' => $columnfamily_name));
 		endforeach;
 	?>
-<?php endif; ?>
\ No newline at end of file
+<?php endif; ?>
+
```
Comment (sebgiroux) [inline views/describe_columnfamily.php]: Feel free to merge all the endif under one php tag

File: views/describe_keyspace.php [hunk 5]
```diff
@@ -7,16 +7,30 @@
...
+	</tr>
+</table>
+
+<div class="well">
 
 <?php echo $added_cf; ?>
 <?php echo $deleted_cf; ?>
 
+<?php
+if ($_GET['view'] == "details"):
```
Comment (sebgiroux) [inline views/describe_keyspace.php]: single quote and isset()

File: views/describe_columnfamily.php (hunk 3 above)
Comment (ghost) [inline views/describe_columnfamily.php]: oops...

**Comment by ghost** [comment]
that should do it

File: describe_keyspace.php [hunk 6]
```diff
@@ -74,10 +74,11 @@
 				for ($i = 0; $i < $nb_cfs; $i++) {
 					$one_cf = $describe_keyspace->cf_defs[$i];
 				
-					$vw_vars['cf_def'] = $one_cf;	
+					$vw_vars['cf_def'] = $one_cf;
 					$vw_vars['columnfamily_name'] = $one_cf->name;				
-					$vw_vars['show_edit_link'] = true;					
-					$vw_vars['list_column_families'] .= getHTML('columnfamily_details.php',$vw_vars);
+					$vw_vars['show_edit_link'] = true;
+					$vw_vars['list_column_families'] .= '';
```
Comment (sebgiroux) [inline describe_keyspace.php]: This line seems useless ? :P

Comment (ghost) [inline describe_keyspace.php]: Oh man, totally forgot that line

**Comment by sebgiroux** [comment]
Awesome, thanks again man!

**Pull request merged**